SU-SWS/stanford_news
# D8CORE-3105: fixing for the no image and external link on a news

# READY FOR REVIEW

# Summary
- When there is a news article with and external source link and no image, there is a double arrow and the font size is wrong.

# Needed By (Date)
- 2/3 - needs to be in the 2.0.0 release.

# Urgency
- High/Blocker

# Steps to Test

1. Pull in this change.
2. Create a couple news articles that have no image and are external links.
3. View the vertical cards on the bottom of a news page and verify they look correct.

# Affected Projects or Products
- Effect the News Module

# Associated Issues and/or People
- D8CORE-3105
- @rmundstock @katrialesser : FYI

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
